### ISSUE TITLE
URL value for set-servicenowauth

### ISSUE THREAD

**Issue by qb411** [Sam-Martin/servicenow-powershell#2]
You might want to provide some more clarity in the msg response.  I initially assumed you wanted the full URL "https://xxxxx.service-now.com/" and ultimately it was just the DNS.  Not trying to nit-pick, but I know I would have found it useful.  I parsed through the modules to find the use of the $url variable and figured it out, so its not anything major.  Hopefully this is useful feedback.

I love the module btw, its really useful!

**Comment by aurbjo** [comment]
I actually did the same thing, or just simply provide an example on the readme page for how to use the URL parameter for the Set-ServiceNowAuth.

Great module! Just started looking at it as i'm doing a integration between SCOM and servicenow, i think you just saved me many many hours of work 👍

**Comment by kamaral75** [comment]
Is there a way to get this working with an instance that has SAML Single Sign-On enabled? We are using the MultiProvider SSO plugin and have an integration with SAML for logins. We had this working previously before we enabled SSO. However, now that we enabled SSO we are getting 401 Unauthorized when attempting to login now. We attempted to use the 'side_door' login page that ServiceNow provides for bypassing the SSO to login to a local account but that still doesn't seem to work. "Set-ServiceNowAuth -url example.service-now.com/side_door.do"
